src/lib/components/Modal/README.mdx

```diff
@@ -74,13 +109,167 @@ See [API](#api) for all available options.
   tells better what happens rather than “OK”.
 
 - Modal **automatically focuses the first form field** by default which allows
-  users to confirm the modal by hitting enter key (requires an action of type
-  `submit`). When no field is found then the first button in footer is focused.
-  To turn this feature off, set the `autofocus` prop to `false`.
+  users to confirm the modal by hitting enter key. When no field is found then
+  the primary button (in footer) is focused. To turn this feature off, set the
+  `autofocus` prop to `false`.
 
 - **Avoid stacking** of modals. While it may technically work, the modal is just
   not designed for that.
 
+## Composition
+
+Modal is decomposed into following components:
+
+- `Modal`
```

Comment:
As for the formatting:

- Component names are not wrapped with `code` in RUI docs to make the output less cluttered (there is already plenty of other `code` elements). This applies to all component name occurences on this page.
- Component names in the first list here could be turned into links to their API—if you agree.

<img width="780" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/175784245-7b4f2111-7521-4c34-bcc0-a8bf83b8f3de.png">

Reply:
Definitely. Please check it in the next round and suggest further changes :)